### ISSUE TITLE
[Expressive modal]: QA: JAWS announces the dialog label as "close modal title"

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9424]
### Description

Page Element:
Screen reader Extraneous announcement for the dialog label

Issue Description:
JAWS announces the dialog label as ""close modal title"" "

Expected:
Screen reader is expected to identify only the dialog label. Here the Word "close" should be removed from the screen reader announcement

Impact:
If the label is incorrect or inadequately descriptive, it affects the screen reader user’s ability to understand the purpose of the control.   

### Component(s) impacted

Expressive Modal

### Browser

Firefox

### Carbon for IBM.com version

v1.39.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

Web component Canary- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components Web component Staging- https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
